### ISSUE TITLE
Giphy 1.2.6 Cannot be constructed because no public initializers

### ISSUE THREAD

**Issue by m1entus** [Giphy/giphy-ios-sdk-ui-example#65]
![Zrzut ekranu 2020-04-27 o 11 08 50](https://user-images.githubusercontent.com/1565368/80354893-a2adb400-8877-11ea-8cb1-5ffd9faccb0d.png)

**Comment by cgmaier** [comment]
hey @m1entus thanks for trying out the new release. a few qs: 
- did you install via Cocoapods or Carthage? if Cocoapods, does your Podfile have the `use_frameworks!` flag? 
- what Xcode version are you using? 
- are you seeing any other compilation warnings / errors?

**Comment by m1entus** [comment]
Hey, yes i use_frameworks i am using newes Cocoapods, have same issue with GPHMediaView probably you have not defined init as a open/public and it is visible only internally so after installing framework i cannot use it.

**Comment by cgmaier** [comment]
what version of Xcode? 

I've tested 1.2.6 and am not able to replicate the issue. The `GiphyViewController` and `GPHMediaView` are publicly accessible. 

It seems like there may be an issue with how you're linking the library. I recommend uninstalling the Pod, reinstalling the pod, close and reopen Xcode, delete derived data, clean etc. and build. 

your Podfile should look something like this : 
```
use_frameworks!

target "YourAppTarget" do 
pod 'Giphy' 
end
```

**Comment by cgmaier** [comment]
hey @m1entus let me know if you happen to have an update! would be great to help resolve this

**Comment by madegraphy** [comment]
Hey @cgmaier, I ran into the same issue on Xcode 11.3. It even goes down to v1.2.4. However, using the latest SDK release with Xcode 11.4 works just fine. Hope this helps.

**Comment by Curius-App** [comment]
Facing the same issue when I upgraded from 1.2.4 to 1.2.6. @cgmaier My pod structure is exactly the same as you mentioned. How to resolve this?

**Comment by cgmaier** [comment]
Hey guys sorry the confusion. I finally re-downloaded Xcode 11.3.1 and confirm the issue is still there.
 
very weird to me that works in Xcode 11.1 and Xcode 11.4 but not 11.3.1

**Comment by DhavalGevariya** [comment]
Hello, Anyone have Solution of this issue..

**Comment by cgmaier** [comment]
hey @DhavalGevariya can upgrade to the latest Xcode?

**Comment by DhavalGevariya** [comment]
Hey Christopher

Yes I have Update Xcode

**Comment by cgmaier** [comment]
hey all, i believe i resolved this with version 1.2.7 

thanks so much for your help! 

cc @DhavalGevariya, @Curius-App, @madegraphy, @m1entus

**Comment by madegraphy** [comment]
Thank you very much @cgmaier! Appreciate your quick response.

**Comment by madegraphy** [comment]
I can confirm that the issue is gone, at least on my side.

**Comment by cgmaier** [comment]
thanks so much for testing @madegraphy !

**Comment by DhavalGevariya** [comment]
@cgmaier Finally, issue fixed in new Version 1.2.7

thank u very much.
